CHANGELOG.md

```diff
@@ -1,168 +1,259 @@
-- Amend README to clarify default deploy module
-- modulesync with latest Vox Pupuli defaults
-
-## 2016-10-04 - Release 3.0.0
-### Summary
-- Change pool_test_on_borrow default to false in init class :: As this alters init class, requires MAJOR release
-- Add access log format variable to init :: as this also hits init class, MAJOR release
-- Update README
-
-### Fixes
-- Fix Download - version 7.1.9 and up
-- Incompatible listeners server.xml > version 7
-- Fix missing pool_test_on_borrow in dbconfig.mysql.xml.erb
-
-### Improvements
-- modulesync with latest Vox Pupuli defaults
-
-## 2016-09-08 - Release 2.1.0
-### Summary
-- modulesync with latest Vox Pupuli defaults
-
-## Improvements
-- Add hiera support for catalina_opts
-
-### Fixes
-- Service status is not working when using init scripts
-- Fix JIRA 7.1.x download URL
-
-## 2016-05-26 - Release 2.0.0
-### Summary
-- We dropped ruby1.8 support!
-- mkrakowitzer-deploy got replaced by puppet-archive.
-- The jira::facts class is included by default. You may get a Error: Duplicate declaration: Class[jira::facts] is already declared
-- We added JIRA 7 Support
-
-### Improvements
-- Use defined function to test for the existence of the fact ::jira_ver
-- Added hieradata examples for Oracle DB backend
-- Added containment for mysql_connector class
-- Support STRICT_VARIABLES=yes
-- Paramaterizing more tomcat settings
-- Added schema parameter for dbconfig
-- Added configuration option to enable or disable websudo.
-- Added tests and updated README.md
-- Support configuration of all properties in jira-config.properties
-- Added crowd single sign-on functions as described by atlassian wiki
-- Added ability to change tomcatShutdownPort
-
-### Fixes
-- Fix to address Amazon Ami configuration
-- Avoid duplicate scheme declaration when using proxy with SSL
-- Fix jira::mysql_connector_url
-
-
-##2015-08-05 - Release 1.3.0
-###Summary
-- Add SSL Support #76
-- Resolve issue where rake lint tasks always exited with 0 status
-- added new parameter disable_notifications in relation to setenv.sh
-- Add examples
-
-
-##2015-04-25 - Release 1.2.1
-###Summary
-- Update metadata, README, CHANGELOG to point to new namespace.
-- Update .travis.yml to auto deploy tagged versions
-
-
-##2015-04-16 - Release 1.2.0
-- Issue #51 Make the jira users shell configurable
-- Add a notify and subscribe options to the jira service
-- Added oracle to db input validator
-- Turn on the AJP connector based on a hiera or puppet-variable lookup
-- Added Microsoft SQL Server support
-- Include jira::facts class by default.
-- Adding new feature generating a content.xml configuration
-- Add a notify and subscribe options to the jira service
-- Provide ability to set the bind address of Tomcat via the jira::tomcatAddress parameter
-- Bump jira version to 6.4.1
-
-Thanks to Scott Searcy, Oliver Bertuch, Paul Geringer, Eric Shamow, William Lieurance, Doug Neal for their contributions.
-
-
-##2014-01-21 - Release 1.1.5
```

Comment:
every release up to 1.2.0 is missing in the new CHANGELOG.md. Those entries should be moved into a HISTORY.md with an adjusted title (it has to match the style from the github-changelog-generator one). Afterwards regenerate the CHANGELOG.md.